src/commands/send_file.rs

```diff
@@ -16,9 +16,12 @@ impl Command for SendFileCommand {
         "send"
     }
 
-    fn parse_params(&self, params: &[&str]) -> Result<Box<dyn Action>> {
-        let file_path = params.get(0).ok_or("No file specified")?;
-        match SendFile::new(file_path) {
+    fn parse_params(&self, param: Option<&str>) -> Result<Box<dyn Action>> {
+        let param = param.ok_or("No file specified")?;
+        let words = shellwords::split(param)?;
```

Comment:
I like a lot the `shellwords` way of parsing commands :)

Only one point here. I think that this line and the previous one is something that any command should do. Could it be made before calling this function and pass the words from shellworlds directly as param of `parse_params`?

Reply:
Updated. @lemunozm